integration-tests/test-addons/addon-with-jandex/src/main/java/org/vaadin/jandex/HelloWorldJandex.java

```diff
@@ -0,0 +1,18 @@
+package org.vaadin.jandex;
+
+import com.vaadin.flow.component.Tag;
+import com.vaadin.flow.component.dependency.JsModule;
+import com.vaadin.flow.component.dependency.NpmPackage;
+import com.vaadin.flow.component.littemplate.LitTemplate;
+
+@Tag("hello-world-jandex")
+@NpmPackage(value = "@axa-ch/input-text", version = "4.3.11")
+@JsModule("./src/hello-world-jandex.ts")
+public class HelloWorldJandex extends LitTemplate {
```

Comment:
As there is no usage of `LitTemplate`features this could just be a `Component` to make it lighter

Reply:
done